tests/unit/test_retry.py

```diff
@@ -59,6 +59,16 @@ def test_w_requests_connection_error(self):
         exc = ValueError("testing")
         self.assertFalse(self._call_fut(exc))
 
+    def test_w_stdlib_connection_error(self):
+        from google.cloud.storage import retry
+
+        try:
+            exc = ConnectionError()
+            self.assertTrue(self._call_fut(exc))
+            self.assertTrue(ConnectionError in retry._RETRYABLE_TYPES)
+        except NameError:
+            pass
+
```

Comment:
Let's mark this testcase as skipped on Python2.  Also, we don't really need to make the assertion about the error being in the list, so e.g.:
```suggestion
    @unittest.skipUnless(
        "ConnectionError" in __builtins__.__dict__,
        "No builtin 'ConnectionError' in Python 2",
    )
    def test_w_stdlib_connection_error(self):
        exc = ConnectionError()
        self.assertTrue(self._call_fut(exc))
```

Reply:
+1 to this.